types/openrct2/test/openrct2-tests.example.ts

```diff
@@ -136,8 +136,8 @@ function open() {
                     ["", ""],
                     sep("VehicleObject"),
                     ["Rotation frame mask:", vehicleObject.rotationFrameMask.toString()],
-                    ["Num. of vertical frames:", vehicleObject.numVerticalFrames.toString()],
-                    ["Num. of horizontal frames:", vehicleObject.numHorizontalFrames.toString()],
+                    ["Num. of vertical frames:", vehicleObject.numVerticalFramesOverride.toString()],
+                    ["Num. of horizontal (base) frames:", vehicleObject.baseNumFrames.toString()],
```

Comment:
I'm 99% sure this is the intended replacement for `numHorizontalFrames` but I'm not really sure? Did this just get removed or was this a good guess? :)

Reply:
Not sure, you can just remove them if you like. This example file is not exhaustively testing every API and every property anyway.